CONTRIBUTING.md

```diff
@@ -66,6 +66,22 @@ To reformat:
 yarn format
 ```
 
+To reformat incremental changes:
+
+```
+yarn prettier --write .prettierrc.json
+```
+
+## Guide generation
+
+The [Concepts](https://docs.temporal.io/docs/temporal/#) and [Application development](https://docs.temporal.io/docs/application-development-guide) guides are generated through the [index.js](guide-gen/index.js) script.
```

Comment:
Good call.

Maybe add something about the guides and the guide generation tooling to be something new and in a sort of alpha state.

And add that the guides should not be added to directly.

Add to source files first, then run `yarn gen` -